### PR TITLE
redesign scheme for backend

### PR DESCRIPTION
**Pull request by mikemangas**
- redesign scheme for database - the majority of properties are based at the overpass api.

### CODE REVIEW

**Review by patrikglanemann — COMMENTED** [review]
Nice structure for the schema. I like that you thought about accessibility for the playgrounfs and included the key wheelchair.

**Comment by ScienceMaster7** [comment]
If I want to view your deployment I get an Application error. If this also happens to you, you should check your deployment logs to figure where the problem might be.

**Comment by mikemangas** [comment]
> If I want to view your deployment I get an Application error. If this also happens to you, you should check your deployment logs to figure where the problem might be.

Thanks Benedikt! I need to check that!

File: models/Playground.js [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
+    },
+  },
+  geometry: {
+    type: {
+      type: String,
+    },
+    coordinates: {
+      type: Array,
     },
   },
```
[the remark this answers is not in the document]
Comment (mikemangas) [reply models/Playground.js:45]: amazing! Thanks for the tipp!!!

File: models/Playground.js [hunk 2]
```diff
@@ -1,32 +1,50 @@
...
-const playGroundSchema = new Schema(
-  {
-    title: {
+const playGroundSchema = new Schema({
+  checkedIn: {
+    type: Number,
+    default: 0,
+  },
+  type: {
+    type: String,
```
[the remark this answers is not in the document]
Comment (mikemangas) [reply models/Playground.js:10]: merci! i will check that out!

**Pull request merged**